dotCMS/src/main/java/com/dotmarketing/servlets/BinaryExporterServlet.java

```diff
@@ -529,7 +533,16 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws Servl
 					Logger.warn(this, e + " Error for = " + req.getRequestURI() + (req.getQueryString() != null?"?"+req.getQueryString():"") );
 					Logger.debug(this, "Error serving asset = " + req.getRequestURI() + (req.getQueryString() != null?"?"+req.getQueryString():""), e);
 
-				} 
+				} finally{
+					if(to!=null){
+						try{
+							to.close();
+						}
+						catch(Exception e){

```

Comment:
You can use CloseUtils to do something like

finally {
   closeQuietly(to);
}